src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/OptionsBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,312 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.Extensions.Hosting.Tests
+{
+    public class OptionsBuilderExtensionsTests
+    {
+        public static IHostBuilder CreateHostBuilder(Action<IServiceCollection> configure)
+        {
+            return new HostBuilder().ConfigureServices(configure);
+        }
+
+        [Fact]
+        public void ValidateOnStart_NullOptionsBuilder_Throws()
+        {
+            var hostBuilder = CreateHostBuilder(services =>
```

Comment:
Is this needed? Why not just call

`OptionsBuilderExtensions.ValidateOnStart(null)`?